requirements/dev.txt

```diff
@@ -298,7 +299,7 @@ text-unidecode==1.3
     #   python-slugify
 toml==0.10.2
     # via ipdb
-tomli==1.2.3
+tomli==2.0.0
```

Comment:
I had to bump this manually to get this to compile. Not sure who the culprit was.